kolibri/auth/constants/role_kinds.py

```diff
@@ -3,14 +3,12 @@
 """
 from __future__ import unicode_literals
 
-from django.utils.translation import ugettext_lazy as _
-
 ADMIN = "admin"
 COACH = "coach"
 ASSIGNABLE_COACH = "classroom assignable coach"
 
 choices = (
-    (ADMIN, _("Admin")),
-    (COACH, _("Coach")),
-    (ASSIGNABLE_COACH, _("Classroom Assignable Coach")),
+    (ADMIN, "Admin"),
+    (COACH, "Coach"),
+    (ASSIGNABLE_COACH, "Classroom Assignable Coach"),
```

Comment:
Is it possible to make this just

```python
choices = (
    ADMIN,
    COACH,
    ASSIGNABLE_COACH,
)```

or does our implementation depend on the structure of the constants?

Reply:
This is required by Django for Model Field choices.